test/mapping/recurrence_tests.jl

```diff
@@ -1,8 +1,44 @@
 using Attractors
 using Test
+using OrdinaryDiffEq:Vern9
 using Random
 using Statistics
 
+@testset "Density of attractors" begin
+    function test_morrislecar(; stop_at_dt, store_once_per_cell)
+        df = Systems.morris_lecar(; I = 0.115)
+        diffeq = (reltol = 1e-9,  alg = Vern9())
+        xg = yg = range(-1,1,length = 20000)
+        mapper = AttractorsViaRecurrences(df, (xg, yg);
+                mx_chk_fnd_att = 5000,
+                mx_chk_loc_att = 1000,
+                stop_at_dt,
+                store_once_per_cell,
+                sparse = true,
+                Δt = 0.1,
+                Ttr = 10,
+                )
+
+        sampler, = Attractors.statespace_sampler(Random.MersenneTwister(1);
+            min_bounds = [-0.5, -1], max_bounds = [0.5, 1])
```

Comment:
use these bounds: ` min_bounds = [-0.5, 0], max_bounds = [0.5, 1])`

The second variable should always be positive (for some negative initial conditions it diverges, I don't know why)